lib/dialogs/about/index.tsx

```diff
@@ -134,8 +131,4 @@ export class AboutDialog extends Component<Props> {
   }
 }
 
-const mapDispatchToProps: S.MapDispatch<DispatchProps> = {
-  closeDialog,
-};
-
-export default connect(null, mapDispatchToProps)(AboutDialog);
+export default AboutDialog;
```

Comment:
we can also pass in an optional `closeDialog` through `OwnProps` and prefer that to the connected props, leaving the normal connected interface

```ts
type OwnProps = {
	closeDialog?: () => any;
}

const mapDispatchToProps: S.MapDispatch<DispatchProps, OwnProps> = (dispatch, props) => ({
	closeDialog: () => dispatch( props.closeDialog?.() ?? closeDialog() ),
})
```